r2i/classification.cc

```diff
@@ -0,0 +1,51 @@
+/* Copyright (C) 2020 RidgeRun, LLC (http://www.ridgerun.com)
+ * All Rights Reserved.
+ *
+ * The contents of this software are proprietary and confidential to RidgeRun,
+ * LLC.  No part of this program may be photocopied, reproduced or translated
+ * into another programming language without prior written consent of
+ * RidgeRun, LLC.  The user is free to modify the source code after obtaining
+ * a software license from RidgeRun.  All source code changes must be provided
+ * back to RidgeRun without any encumbrance.
+*/
+
+#include "classification.h"
+
+namespace r2i {
+
+Classification::Classification() {
+  this->type = InferenceOutputType::CLASSIFICATION;
+}
+
+Classification::~Classification() {
+  this->labels.clear();
+}
+
+RuntimeError Classification::SetLabels(std::vector< ClassificationInstance >
+                                       labels) {
+  this->labels = labels;
+  return RuntimeError();
+}
+
+std::vector< ClassificationInstance > Classification::GetLabels() {
+  return this->labels;
+}
+
+ClassificationInstance Classification::GetRank1() {
```

Comment:
I think this should be removed. This a PostProcessing role.

Reply:
Once you have the vector of tuples, this is a repetitive logic that can be factorized in here.